docs/source/usage/plugins/particleCalorimeter.rst

```diff
@@ -75,19 +75,18 @@ Detection of outgoing particles
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 In order to detect particles which are leaving the global simulation volume, the calorimeter plugin has to be triggered whenever this happens.
-This is established by adding the ``GuardHandlerCallPlugins`` template argument to ``ParticleDescription`` (see :ref:`speciesDefinition.param <usage-params-core>`):
+This is established by adding the alias ``boundaryCondition< GuardHandlerCallPlugins >`` to the particle species flags (see :ref:`speciesDefinition.param <usage-params-core>`):
```

Comment:
I would maybe call it `onParticleLeave` / `whenLeavingSimVolume` or sth of the sort - just like the optional plugin member function it activates.
The name `boundaryCondition` may be a bit misleading.

Reply:
nope, we can make this the beginning of the initial boundary condition handling beyond absorbing/periodic, but the interface needs to be refactored.

By default, we will make absorbing boundary conditions call the plugin guard handler.